diff --git a/experimental/td3_dmc_vanilla_dynamics.py b/experimental/td3_dmc_vanilla_dynamics.py
index 33fafad..9b4c9e0 100644
--- a/experimental/td3_dmc_vanilla_dynamics.py
+++ b/experimental/td3_dmc_vanilla_dynamics.py
@@ -221,8 +221,9 @@ poetry run pip install "stable_baselines3==2.0.0a1"
 
     device = torch.device("cuda" if torch.cuda.is_available() and args.cuda else "cpu")
 
-
-    with open('/home/ubuntu/pywork/Plasticine/plasticine/standard/frictions', 'rb+') as f:
+    current_file_path = os.path.abspath(__file__)
+    current_directory = os.path.dirname(current_file_path)
+    with open(current_directory+'/frictions', 'rb+') as f:
             frictions = pickle.load(f)
     friction_number = 0
     
